### PR TITLE
add solutoin v1.0

### PR DESCRIPTION
**Pull request by HokageHEL**
[DEMO LINK](https://hokagehel.github.io/react_goods-selector-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +15,81 @@ export const goods = [
...
+        <tbody>
+          {goods.map((good) => {
+            const isGoods = x => x === value;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+            return (
+              <tr data-cy="Good" className={`${isGoods(good) && 'has-background-success-light'}`}>
```
Comment (denys-danyliuk) [inline src/App.jsx:53]: add key
```suggestion
              <tr 
                data-cy="Good" 
                className={`${isGoods(good) && 'has-background-success-light'}`}
              >
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +15,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```
Comment (denys-danyliuk) [inline src/App.jsx:19]:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```

File: src/App.jsx [hunk 3]
```diff
@@ -14,70 +15,81 @@ export const goods = [
...
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isGoods = x => x === value;
```
Comment (denys-danyliuk) [inline src/App.jsx:50]:
```suggestion
            const isSelected = x => x === value;
```
also use more meaningful variable names

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 4]
```diff
@@ -14,70 +15,84 @@ export const goods = [
...
+            const isSelected = x => x === selectedGood;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={`${isSelected(good) && 'has-background-success-light'}`}
```
Comment (etojeDenys) [inline src/App.jsx:55]: let's use classnames package for such cases
image

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 5]
```diff
@@ -14,70 +16,86 @@ export const goods = [
...
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = x => x === selectedGood;
```
Comment (VitaliyBodnar) [inline src/App.jsx:51]: can be less
```suggestion
            const isSelected = good === selectedGood;
```

File: src/App.jsx [hunk 6]
```diff
@@ -14,70 +16,86 @@ export const goods = [
...
+                  {
+                    isSelected(good)
+                      ? (
+                        <button
+                          data-cy="RemoveButton"
+                          type="button"
+                          className="button is-info"
+                          onClick={() => {
+                            setselectedGood('');
+                          }}
```
Comment (VitaliyBodnar) [inline src/App.jsx:70]: let's align your code
check other places 
```suggestion
                          onClick={() =>setselectedGood('')}
```

File: src/App.jsx [hunk 7]
```diff
@@ -14,70 +16,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState('Jam');
```
Comment (VitaliyBodnar) [inline src/App.jsx:20]:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

**Review by lerastarynets — APPROVED** [review]
You've got this, keep pushing forward!

**Pull request closed without merging**